pages.es/linux/conky.md

```diff
@@ -0,0 +1,28 @@
+# conky
+
+> Monitor de sistema ligero para X.
+> Más información: <https://github.com/brndnmtthws/conky>.
+
+- Ejecutar con la configuración por defecto:
+
+`conky`
+
+- Crear una nueva configuración por defecto:
+
+`conky -C > ~/.conkyrc`
+
+-Ejecuta conky con un archivo de configuración concreto:
+
+`conky -c {{path/to/config}}`
```

Comment:
Is this in Spanish?

Reply:
No, it definitely isn't. :laughing: It's something like `camino/a/la/config`.

Reply:
> Is this in Spanish?

Haha, no, it isn't Spanish. It was a little slip I had translating.

> No, it definitely isn't.  It's something like `camino/a/la/config`.

Yes, it is something similar. It should be changed to `{{ruta/a/la/configuración}}`.

Reply:
edbcb9d solved this review, @sbrl, @zdroid. Thank you for point this.